### PR TITLE
optimize some docs.

### PR DESCRIPTION
**Pull request by jialigit**
fix some doc for aad migration guide.

### CODE REVIEW

File: docs/src/main/asciidoc/_migration-guide-for-4.0.adoc [hunk 1]
```diff
@@ -295,85 +297,157 @@ version 3 of link:https://search.maven.org/artifact/com.azure.spring/azure-sprin
...
+|org.springframework.boot:spring-boot-starter-webflux
+|Please add this dependency to your project if needed.
 |===
 
 [#configuration-spring-cloud-azure-starter-active-directory]
-===== SDK Configuration Changes
-
-. All configuration property names changed the prefix from `azure.activedirectory` to `spring.cloud.azure.active-directory`.
-. New property `spring.cloud.azure.active-directory.enabled=true` is added to allow enable / disable AAD related features. The default value is false.
+===== SDK Configuration Changes(Added/Removed/Changed)
```
Comment (saragluna) [inline docs/src/main/asciidoc/_migration-guide-for-4.0.adoc:323]: I don't think we need `(Added/Removed/Changed)` here

File: docs/src/main/asciidoc/_migration-guide-for-4.0.adoc [hunk 2]
```diff
@@ -295,85 +297,157 @@ version 3 of link:https://search.maven.org/artifact/com.azure.spring/azure-sprin
...
-|===
+.For modern:
+spring.cloud.azure.active-directory.profile.environment.microsoft-graph-endpoint=`https://graph.microsoft.com/` +
+spring.cloud.azure.active-directory.user-group.use-transitive-members=`false`
+====
 
-NOTE: `azure.activedirectory.graph-membership-uri` has been replaced by 2 properties: `spring.cloud.azure.active-directory.profile.environment.microsoft-graph-endpoint` and `spring.cloud.azure.active-directory.user-group.use-transitive-members`. The first property used to specify the host name, and the second property used to specify the url path: `v1.0/me/memberOf` or `v1.0/me/transitiveMemberOf`. Here are some examples:
+.Case 2
+====
+.For legancy:
```
Comment (saragluna) [inline docs/src/main/asciidoc/_migration-guide-for-4.0.adoc:444]: legacy

File: docs/src/main/asciidoc/_migration-guide-for-4.0.adoc [hunk 3]
```diff
@@ -295,85 +297,157 @@ version 3 of link:https://search.maven.org/artifact/com.azure.spring/azure-sprin
...
+* spring.cloud.azure.active-directory.profile.environment.microsoft-graph-endpoint
+* spring.cloud.azure.active-directory.user-group.use-transitive-members
+
+NOTE: `azure.activedirectory.graph-membership-uri`'s function has been replaced by 2 properties: `spring.cloud.azure.active-directory.profile.environment.microsoft-graph-endpoint` and `spring.cloud.azure.active-directory.user-group.use-transitive-members`. The first property is used to specify the host name, and the second a flag for using the url path: `v1.0/me/memberOf` or `v1.0/me/transitiveMemberOf`.
+
+Here are some examples of migration:
+
+.Case 1
 ====
+.For legancy:
```
Comment (saragluna) [inline docs/src/main/asciidoc/_migration-guide-for-4.0.adoc:434]: legacy

File: docs/src/main/asciidoc/_migration-guide-for-4.0.adoc [hunk 4]
```diff
@@ -1,6 +1,7 @@
 
 [#migration-guide-for-4-0]
 == Migration Guide for 4.0
+:icons: font
```
Comment (saragluna) [inline docs/src/main/asciidoc/_migration-guide-for-4.0.adoc:4]: what's this for

**Review by saragluna — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/_migration-guide-for-4.0.adoc (hunk 2 above)
Comment (jialigit) [reply docs/src/main/asciidoc/_migration-guide-for-4.0.adoc]: good catch

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**